packages/core/src/Site/index.js

```diff
@@ -107,8 +107,9 @@ const ABOUT_MARKDOWN_DEFAULT = '# About\n'
 const MARKBIND_LINK_HTML = `<a href='${MARKBIND_WEBSITE_URL}'>MarkBind ${MARKBIND_VERSION}</a>`;
 
 class Site {
-  constructor(rootPath, outputPath, onePagePath, forceReload = false,
-              siteConfigPath = SITE_CONFIG_NAME, dev, backgroundBuildMode, postBackgroundBuildFunc) {
+  constructor(rootPath, outputPath, onePagePath, dev,
```

Comment:
With this change, I would think that the calls to this constructor, via `new Site(...` should also be updated, which I think there are a few usages in `packages\cli\index.js`. May I check if you are aware and if there's any reason that those are not updated?

I guess our existing test cases are not comprehensive enough that the CI is still passing without the above changes 😓

Reply:
I used my IDE refactor method (to refactor all usages and I guess it was not working) and since it passed all test cases I tot it was working fine.
Sorry. I will look into this.